public/app/features/admin/UserAdminPage.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { PureComponent } from 'react';
+import { hot } from 'react-hot-loader';
+import { connect } from 'react-redux';
+import { NavModel } from '@grafana/data';
+import { getNavModel } from 'app/core/selectors/navModel';
+import { getRouteParamsId } from 'app/core/selectors/location';
+import config from 'app/core/config';
+import Page from 'app/core/components/Page/Page';
+import { UserProfile } from './UserProfile';
+import { UserPermissions } from './UserPermissions';
+import { UserSessions } from './UserSessions';
+import { UserLdapSyncInfo } from './UserLdapSyncInfo';
+import { StoreState, UserDTO, UserOrg, UserSession, SyncInfo, UserAdminError } from 'app/types';
+import {
+  loadAdminUserPage,
+  loadUserProfile,
+  loadUserOrgs,
+  loadUserSessions,
+  revokeSession,
+  revokeAllSessions,
+  loadLdapSyncStatus,
+  updateUser,
+  disableUser,
+  enableUser,
+  deleteUser,
+} from './state/actions';
+import { UserOrgs } from './UserOrgs';
+
+interface Props {
+  navModel: NavModel;
+  userId: number;
+  user: UserDTO;
+  orgs: UserOrg[];
+  sessions: UserSession[];
+  ldapSyncInfo: SyncInfo;
+  isLoading: boolean;
+  error: UserAdminError;
+
+  loadAdminUserPage: typeof loadAdminUserPage;
+  loadUserProfile: typeof loadUserProfile;
+  loadUserOrgs: typeof loadUserOrgs;
+  loadUserSessions: typeof loadUserSessions;
+  loadLdapSyncStatus: typeof loadLdapSyncStatus;
```

Comment:
these are not used anymore

Reply:
Yea, I have to refactor and remove unused components, state and actions.